integration-test/1627-modify-zoom-bicycle-theatre-water-tower.py

```diff
@@ -0,0 +1,253 @@
+                'id': 260354461,
+                'kind': 'bicycle',
+                # should be in z15 tile, so min_zoom between 15 and 16
+                'min_zoom': lambda z: 15 <= z < 16,
+            })
+
+    def test_bicycle_shop_small_way(self):
+        import dsl
+
+        z, x, y = (16, 10476, 25332)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/way/264534357
+            dsl.way(264534357, dsl.box_area(z, x, y, 362), {
+                'addr:city': 'San Francisco',
+                'addr:housenumber': '858',
+                'addr:postcode': '94117',
+                'addr:street': 'Stanyan Street',
+                'building': 'yes',
+                'height': '5',
+                'name': 'American Cyclery Too',
+                'operator': 'American Cyclery',
+                'service:bicycle:pump': 'yes',
+                'service:bicycle:repair': 'yes',
+                'service:bicycle:retail': 'yes',
+                'service:bicycle:second_hand': 'yes',
+                'shop': 'bicycle',
+                'source': 'openstreetmap.org',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'pois', {
+                'id': 264534357,
+                'kind': 'bicycle',
+                # min_zoom between 16 and 17, less than the node at z17
+                'min_zoom': lambda z: 16 <= z < 17,
+            })
+
+
+class TheatreTest(FixtureTest):
+
+    def test_theatre_node(self):
+        import dsl
+
+        z, x, y = (16, 10483, 25329)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/node/358805392
+            dsl.point(358805392, (-122.411371, 37.782168), {
+                'amenity': 'theatre',
+                'ele': '14',
+                'gnis:county_id': '075',
+                'gnis:created': '01/01/1995',
+                'gnis:feature_id': '1657186',
+                'gnis:state_id': '06',
+                'name': 'Market Street Theatre',
+                'source': 'openstreetmap.org',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'pois', {
+                'id': 358805392,
+                'kind': 'theatre',
+                'min_zoom': 17,
+            })
+
+    def test_theatre_medium_way(self):
+        import dsl
+
+        z, x, y = (16, 10483, 25330)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/way/35115840
+            dsl.way(35115840, dsl.box_area(z, x, y, 4782), {
+                'amenity': 'theatre',
+                'building': 'yes',
+                'height': '46 m',
+                'name': 'Orpheum Theatre',
+                'source': 'openstreetmap.org',
+                'wikidata': 'Q7103971',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'pois', {
+                'id': 35115840,
+                'kind': 'theatre',
+                'min_zoom': lambda z: 16 <= z < 17,
```

Comment:
There's probably more work to do here with respect to building volume or other indicators – as this is one of the more significant theatres in SF. But this is good for now.